packages/replay/src/replay.ts

```diff
@@ -148,49 +149,72 @@ export class ReplayContainer implements ReplayContainerInterface {
   }
 
   /**
-   * Initializes the plugin.
-   *
-   * Creates or loads a session, attaches listeners to varying events (DOM,
-   * _performanceObserver, Recording, Sentry SDK, etc)
+   * Initializes the plugin based on configuration options. Should not be called outside of constructor
    */
-  public start(): void {
-    this.setInitialState();
+  public initializeSampling(): void {
```

Comment:
Split start into two methods: `initializeSampling()` and `_initializeRecording()`